docs/src/api/class-browsertype.md

```diff
@@ -102,13 +101,13 @@ This methods attaches Playwright to an existing browser instance.
     `30000` (30 seconds). Pass `0` to disable timeout.
 
 ### param: BrowserType.connect.wsEndpoint
-* langs: java, python
+* langs: java, python, csharp
 - `wsEndpoint` <[string]>
 
 A browser websocket endpoint to connect to.
 
 ### option: BrowserType.connect.headers
-* langs: java, python
+* langs: java, python, csharp
```

Comment:
What's up with BrowserType.connect.slowMo & BrowserType.connect.timeout?

Reply:
This is not implemented as of now so was suggesting we can add this later when we make changes to playwright dotnet. Will implement this feature in follow-up PR.  So,  was thinking when we start developing only then we expose. Right now I don’t see a need of doing it since this is not being used.

Reply:
What are your thoughts? Should I change it?

Reply:
Yes, we only ship full APIs, so having support for slowMo and connection timeout needs to be implemented before shipping it.

Reply:
Updated